lib/remote_command_runner.rb

```diff
@@ -56,4 +56,86 @@ def run(*args)
   rescue Cheetah::ExecutionFailed => e
     raise ExecutionFailed.new(e)
   end
+
+  # Copy a local file to the remote system.
+  #
+  # +source+:: Path to the local file
+  # +destination+:: Path to the remote file or directory. If +destination+ is a
+  #                 path, the same filename as +source+ will be used.
+  # +opts+:: Options to modify the attributes of the remote file.
+  #
+  #          Available options:
+  #          [owner]:: Owner of the file, e.g. "tux"
+  #          [group]:: Group of the file, e.g. "users"
+  #          [mode]:: Mode of the file, e.g. "600"
+  def inject_file(source, destination, opts = {})
```

Comment:
Assignment Branch Condition size for inject_file is too high. [16.76/15]<br>Perceived complexity for inject_file is too high. [8/7]